
uint8 EVENT_BEGIN=0
uint8 EVENT_END=1

float64 timestamp
int32 polarization
string event_type
string[] participants
